### DISCUSSION TITLE
Feature: Multiple Time Systems

### DISCUSSION THREAD

**Discussion by LunamOdium** [AscensionGameDev/Intersect-Engine#1757] [Ideas]
### **Is your feature request related to a problem? Please describe.**
Not a problem. Sorry guys I know I'm posting a lot.
So it would be nice to have mutiple time systems for mutiple maps.
E.G; in my game i have multiple dimensions, I dont want the normal overworld time system in my other dimensions because I want cool ambient times that fit that particular dimension. 

Currently we have to mark our maps as "interior" and set the darkness levels individually. This doesnt allow you to use the time system at all for that particular map.

### **Describe the solution you'd like**
Add the ability to have multiple time systems that can be chosen from the Map Properties(or if none is chosen automatically choose the first time system). 

### **Duplicate Check**

- [x] This post is not a duplicate to the best of my knowledge.

**Comment by PyroTech03** [comment]
This would be great for my game as there's multiple planets.
